src/test/AludelFactoryIntegration.sol

```diff
@@ -61,17 +64,24 @@ contract AludelFactoryIntegrationTest is DSTest {
     }
 
     uint256 public constant BASE_SHARES_PER_WEI = 1000000;
+    uint256 public constant STAKE_AMOUNT = 1 ether;
+    uint256 public constant REWARD_AMOUNT = 10 ether;
 
     AludelInitializationParams private defaultParams;
 
     function setUp() public {
-        cheats = Hevm(HEVM_ADDRESS);
-        owner = cheats.addr(PRIVATE_KEY);
-
-        recipient = cheats.addr(PRIVATE_KEY + 1);
+        vm = Vm(HEVM_ADDRESS);
```

Comment:
Given that the `Utils` file is imported here I think we could do without the `HEVM_ADDRESS` and just call `Utils.vm()`

Reply:
Yeees